cirq/api/google/v2/metrics.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package cirq.api.google.v1;
+
+option java_package = "com.google.cirq.api.google.v2";
+option java_outer_classname = "CalibrationProto";
+option java_multiple_files = true;
+
+// Performance metrics for a quantum processor.
+message Metrics {
+
+  // The time the metrics were collected, in milliseconds since Jan 1, 1970.
```

Comment:
This quantity is ambiguous. Refer to a specific time standard, such as unix time or UTC. Include it in the field name.

Reply:
Unix time is in UTC.  

I've added words to say it is unix time.  I think calling it "unix time" doesn't really help, but labeling the time unit is important.